src/data/registry.xml

```diff
@@ -491,5 +491,76 @@
       <long_name>graupel mass mixing ratio with respect to moist air plus all airborne condensates</long_name>
       <ic_file_input_names>GRAUQM cnst_GRAUQM</ic_file_input_names>
     </variable>
+
+    <!-- cam_in Variables -->
+    <variable local_name="landfrac"
+              standard_name="land_area_fraction"
+              units="fraction" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension</dimensions>
+      <ic_file_input_names>landfrac cam_in_landfrac</ic_file_input_names>
+    </variable>
+
+    <!-- Zhang McFarlane (ZM) Variables -->
+    <!-- Note that no_deep_pbl defaults to false, but needs to be set to true for diag_TKE or UW schemes -->
+    <variable local_name="pblh"
+              standard_name="atmosphere_boundary_layer_thickness"
+              units="m" type="real" kind="kind_phys"
+              allocatable="allocatable">
+      <dimensions>horizontal_dimension</dimensions>
+      <ic_file_input_names>pblh pbuf_pblh</ic_file_input_names>
+    </variable>
+    <variable local_name="delt"
+              standard_name="half_timestep_for_physics"
+              units="s" type="real" kind="kind_phys">
+    </variable>
```

Comment:
Is this variable needed anymore?  I couldn't see any use of it in the CCPP-ized ZM schemes.